cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1449,11 +1441,22 @@ public void initialized() {
             unprocessedOpenedNotifications.add(openedResult);
             return false;
         }
-        notificationOpenedListener.notificationOpened(openedResult);
 
+        notificationOpenedListener.notificationOpened(openedResult, NotificationOpenedActivity.notificationOpenActivity);
```

Comment:
Not needed as per y understanding, we can save `NotificationOpenedActivity` inside `NotificationOpenedCallbackListener`

Reply:
For this and the above comment 

> why not instance of NotificationOpenedCallbackListener?

Previously we have created separate interface for NotificationOpenedCallbackListener
But then we need to those in the .init method 
So you told us that we do not need to modify init method (com/cleverpush/CleverPush.java:398)

So that's why discussed this with Kevin and come out with this solution

But Now if you want to have a separate interface for NotificationOpenedCallbackListener 
Let's maybe connect on call and check together